src/components/Dashboard/RightClick.vue

```diff
@@ -215,8 +215,8 @@ const menuData = computed<RightClickMenuEntryType[]>(() => [
       ...(torrent.value?.tags.length
         ? [
             {
-              text: t('dashboard.right_click.tags.removeall'),
-              action: async () => await removeTags(maindataStore.tags),
+              text: t('dashboard.right_click.tags.remove_all'),
+              action: async () => await removeAllTags(),
```

Comment:
*(Optional)*: You can also pass the function directly without surrounding it in a lambda
```suggestion
              action: removeAllTags,
```